src/main/java/Duke.java

```diff
@@ -20,47 +19,53 @@ public static void main(String[] args) {
+                        ,false);
+                taskList.addTasks(toDoTask);
+                toDoTask.initialiseToDo();
+                System.out.println("Now you have " + taskList.countTaskInList()
+                        + " tasks in the list");
+                System.out.println("______________________________\n");
+                break;
 
-                case "event":
-                    Events eventTask = new Events(
-                            userInput.substring(userInput.indexOf(' ',0), userInput.indexOf('/')),
-                            false,identifyDeadlineCommand(userInput)[1]);
-                    taskList.addTasks(eventTask);
-                    eventTask.initialiseEvent();
-                    System.out.println("Now you have " + taskList.countTaskInList()
-                            + " tasks in the list");
-                    System.out.println("______________________________\n");
-                    break;
+            case "deadline":
+                Deadline deadLineTask = new Deadline(
+                        userInput.substring(userInput.indexOf(' ',0), userInput.indexOf('/'))
+                        ,false,identifyDeadlineCommand(userInput)[1]);
+                taskList.addTasks(deadLineTask);
+                deadLineTask.initialiseDeadline();
+                System.out.println("Now you have " + taskList.countTaskInList()
+                        + " tasks in the list");
+                System.out.println("______________________________\n");
+                break;
 
-                case "list":
-                    System.out.println("______________________________\n");
-                    taskList.listTasks();
-                    System.out.println("______________________________\n");
-                    break;
+            case "event":
+                Events eventTask = new Events(
+                        userInput.substring(userInput.indexOf(' ',0), userInput.indexOf('/')),
+                        false,identifyDeadlineCommand(userInput)[1]);
+                taskList.addTasks(eventTask);
+                eventTask.initialiseEvent();
+                System.out.println("Now you have " + taskList.countTaskInList()
+                        + " tasks in the list");
+                System.out.println("______________________________\n");
+                break;
 
-                case "done":
-                    taskListIndex = Integer.parseInt(identifyUserInput(userInput)[1]);
-                    currentTask = taskList.findTask(taskListIndex);
-                    currentTask.markTaskAsDone();
-                    break;
-                default:
-                    throw new IllegalStateException("Unexpected value: " + userInput);
+            case "list":
+                System.out.println("______________________________\n");
+                taskList.listTasks();
+                System.out.println("______________________________\n");
```

Comment:
multiple lines with the same print line separator in the code, maybe try to make the print line a constant?